usaspending_api/search/v2/views/spending_by_award_count.py

```diff
@@ -125,3 +138,38 @@ def handle_subawards(filters: dict) -> dict:
         results["subcontracts"] = sum([sub["count"] for sub in queryset if sub["award_type"] == "procurement"])
 
         return results
+
+    def query_elasticsearch(self, filters) -> list:
+        filter_query = QueryWithFilters.generate_elasticsearch_query(filters, query_type="awards")
+        s = Search(index=f"{settings.ES_AWARDS_QUERY_ALIAS_PREFIX}*").filter(filter_query)
+
+        s.aggs.bucket(
+            "types",
+            "filters",
+            filters={
+                "contracts": Q("terms", type=["A", "B", "C", "D"]),
```

Comment:
Would be best to leverage a dictionary in usaspending-api/usaspending_api/awards/v2/lookups/lookups.py.
Perhaps `all_awards_types_to_category` since it is is already being imported and used in the postgres business logic. Although, `all_award_types_mappings` might make a nice dict comprehension if I'm not mistaken

```python
filters = {category: Q("terms", type=types) for category, types in all_award_types_mappings.items()}
```

Reply:
fair